CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ## Added
 * Added `_meta` to all entries and assets [#117](https://github.com/contentful/contentful_middleman/issues/117) [#93](https://github.com/contentful/contentful_middleman/issues/93)
+* Added `client_options` option to extension options to be able to control Client instantiation
```

Comment:
Maybe better remove `option` right after `client_options`? as it is important to explain where their are supposed to go.